### PR TITLE
Fix/#103 model smart contract doesnt publish get method on block explorer

### PR DESCRIPTION
**Pull request by hoang-dao**

### CODE REVIEW

File: packages/aloxide-js/src/helpers/validate-network-config.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ export function validateNetworkConfig(config) {
     throw new Error('Missing network config!');
   }
 
-  const requiredKeys = 'name,type,protocol,host,port,chainId,coreToken'.trim().split(',');
+  const requiredKeys = 'name,type,protocol,host,chainId,coreToken'.trim().split(',');
```
Comment (manh-vv) [inline packages/aloxide-js/src/helpers/validate-network-config.ts:6]: It is funny that you use trim() and split() here.

File: packages/bridge/smart-contract-templates/icon/icon_hello/icon_hello.py.hbs [hunk 2]
```diff
@@ -22,12 +22,14 @@ class hello(IconScoreBase):
         return "Hi"
 
     {{#each tables}}
-    @external
+    ################################################
+    # "{{name}}" supported functions
+    ################################################
+
+    @external(readonly=True)
```
Comment (manh-vv) [inline packages/bridge/smart-contract-templates/icon/icon_hello/icon_hello.py.hbs:29]: This update belongs to the bridge scope but there is no commit which mentioned it.

BTW, should we continue to merge this into the main branch? I saw it really hurt the coverage.

File: packages/aloxide-js/src/helpers/validate-network-config.ts (hunk 1 above)
Comment (hoang-dao) [reply packages/aloxide-js/src/helpers/validate-network-config.ts:6]: hmm... we should point the issue when reviewing, "something is funny that..." is an exclamatory phrase and it doesn't show the action I should do here.

File: packages/bridge/smart-contract-templates/icon/icon_hello/icon_hello.py.hbs (hunk 2 above)
Comment (hoang-dao) [reply packages/bridge/smart-contract-templates/icon/icon_hello/icon_hello.py.hbs:29]: I think you mistaken it, the coverage is slightly changed (0.08%) compare to previous commit. Btw, I'll update to fill the gap.

Comment (hoang-dao) [reply packages/bridge/smart-contract-templates/icon/icon_hello/icon_hello.py.hbs:29]: I forgot that the PR https://github.com/lecle/aloxide/pull/110 add a lot of test cases, I'll create another PR for the missing cases.

**Review by manh-vv — APPROVED** [review]
LGTM

**Pull request merged**